scripts/helmcharts/openreplay/charts/utilities/templates/efs-cron.yaml

```diff
@@ -44,7 +43,6 @@ spec:
             volumeMounts:
             - mountPath: /mnt/efs
               name: datadir
-          restartPolicy: Never
           {{- if eq (tpl .Values.efsCleaner.pvc.name . ) "hostPath" }}
```

Comment:
We don't restart, as this is a deletion operation, and don't want to delete all the data if something goes wrong and the operation restarts.

Reply:
`restartPolicy: Never` is already defined here:
https://github.com/openreplay/openreplay/blob/4872f1f8355e05c2904df90538d9a79700a2a040/scripts/helmcharts/openreplay/charts/utilities/templates/efs-cron.yaml#L25

This is a duplicate I'm removing